src/jimgw/prior.py

```diff
@@ -143,6 +144,68 @@ def log_prob(self, z: dict[str, Float]) -> Float:
         return -0.5 * variable**2 - 0.5 * jnp.log(2 * jnp.pi)
 
 
+@jaxtyped(typechecker=typechecker)
+class SimplexBaseDistribution(Prior):
+    """
+    A 2D distribution that is uniformly distributed within the triangle formed by the vertices (0, 0), (1, 0), and (0, 1) being mapped to Logistic Distribution.
+    """
+
+    def __repr__(self):
+        return f"SimplexBaseDistribution(parameter_names={self.parameter_names})"
+
+    def __init__(self, parameter_names: list[str], **kwargs):
+        super().__init__(parameter_names)
+        self.composite = False
+        assert self.n_dim == 2, "SimplexDistribution needs to be 2D distributions"
+        
+    @staticmethod
+    def unit_simplex_transform(x: Float[Array, " n_dim"]) -> Float[Array, " n_dim"]:
+        z1 = jnp.log(x[0] / (1.0 - x[0]))
```

Comment:
The typing now does not agree with the implementation. The typing suggests this works for n dimensional input, but it only work for 2 dimensional input